src/org/jetbrains/plugins/scala/compiler/ScalaHydraCompilerConfigurationPanel.scala

```diff
@@ -37,20 +40,53 @@ class ScalaHydraCompilerConfigurationPanel(project: Project, settings: HydraComp
   userTextField.getDocument.addDocumentListener(documentAdapter)
   passwordTextField.getDocument.addDocumentListener(documentAdapter)
   passwordTextField.addFocusListener(focusListener)
-  hydraVersionComboBox.setItems(Versions.loadHydraVersions)
+  hydraVersionComboBox.setItems(downloadHydraVersions)
   downloadButton.addActionListener((_: ActionEvent) => onDownload())
+  noOfCoresComboBox.setItems(Array.range(1, Runtime.getRuntime.availableProcessors()/2 + 1).map(_.toString))
```

Comment:
The higher number should be at the top (maybe that's already the case, but I'm not sure, which is why I'm making a note here)

Reply:
I wonder if it's worth using a combo box here. We can let people enter whatever number of cores they want, we can just have a default to the number of cores/2.

Reply:
Good point. Also consider that the range should also at least be between 1 and `Runtime.getRuntime.availableProcessors()` (no division by two)

Reply:
If I change the hydra workers combo box into a field, I also have to validate the input because it might not be a number or is a negative number. So I'll look into how the validation is done on a field, but I think is much simpler just to use the combo box.

Reply:
Agreed. Let's keep things simple and use a list box with a range from 1 to <number of cores> and set the default to `number of cores/2`. Also, the list box should display the higher number at the top.